pages/_en-US/twilightmenu/custom-dsi-3ds-sfx.md

```diff
@@ -70,6 +70,9 @@ To export in the correct format:
 1. Set the output name to `bgm.pcm.raw` and click `Save`
 1. Click `OK` to the metadata editing
 
-Now you have a `bgm.pcm.raw` file that can be copied to the `sound` subfolder in your theme folder.
+---
+
+Once both of these files are ready to be used, place them in the `sound` subfolder for the skin.
+- Examples: `sd:/_nds/TWiLightMenu/dsimenu/themes/[theme name]/sound/sfx.bin` & `sd:/_nds/TWiLightMenu/dsimenu/themes/[theme name]/sound/bgm.pcm.raw`
```

Comment:
```suggestion
Once both of these files are ready to be used, place them in the `sound` subfolder for the skin. For example:
- `sd:/_nds/TWiLightMenu/dsimenu/themes/[theme name]/sound/sfx.bin`
- `sd:/_nds/TWiLightMenu/dsimenu/themes/[theme name]/sound/bgm.pcm.raw`
```
I think this'd look better like this, this also needs to change to bgm.wav iirc but I'm not suggesting that here as I haven't suggested the other relevant changes.